src/script/components/panel/UserDetails.tsx

```diff
@@ -67,19 +66,14 @@ const UserDetailsComponent: React.FC<UserDetailsProps> = ({
   return (
     <div className="panel-participant">
       <div className="panel-participant__head">
-        {teamState.isInTeam(participant) ? (
-          <AvailabilityState className="panel-participant__head__name" user={participant} dataUieName="status-name">
-            {renderParticipantBadges?.(participant)}
-          </AvailabilityState>
-        ) : (
-          <h2
-            className="panel-participant__head__name"
-            data-uie-name="status-name"
-            css={user.isAvailable ? undefined : {color: 'var(--gray-70)'}}
-          >
-            <UserName user={participant} />
-          </h2>
-        )}
```

Comment:
This is the bug fix: this component would not render the verification badges if the user is not part of the same team